### PR TITLE
update weight default in route to service

### PR DESCRIPTION
**Pull request by laurafitzgerald**
**What**
Update route default to 100.

Based on my investigations the default value for the Route -> spec -> to -> weight is 100 when unspecified.

### CODE REVIEW

File: route/v1/types.go [hunk 1]
```diff
@@ -90,7 +90,7 @@ type RouteSpec struct {
 	Path string `json:"path,omitempty" protobuf:"bytes,2,opt,name=path"`
 
 	// to is an object the route should use as the primary backend. Only the Service kind
-	// is allowed, and it will be defaulted to Service. If the weight field (0-256 default 1)
+	// is allowed, and it will be defaulted to Service. If the weight field (0-256 default 100)
```
Comment (soltysh) [inline route/v1/types.go]: Yup, https://github.com/openshift/origin/blob/23d4f6fb4cc71e89f2a5a5972d0ea5e406b551ad/pkg/route/apis/route/v1/defaults.go#L19-L22

**Comment by soltysh** [comment]
@laurafitzgerald I think you might be missing running `make generate` to update generated files.

**Comment by laurafitzgerald** [comment]
@soltysh yes I didn't do that. I'm new to operators. Will run now and push the change.

**Comment by laurafitzgerald** [comment]
@soltysh `make generate` is looking for a specific version of protoc `= 3.0.x`. I've installed the latest as I assumed it would be happy with `=>` but it still doesn't like it. Is also mentions that this step can be skipped. What's the correct way to proceed? Skip it or go to the specific version? Is it possible that newer versions would work but the check is very specific?

**Comment by laurafitzgerald** [comment]
Looks like [this import](https://github.com/openshift/api/blob/master/apps/v1/generated.proto#L10) is not used

**Comment by soltysh** [comment]
> EDIT: generated changes now pushed: I also see some additional files in the `vendor` folder. Not sure if I should include those?

You don't touch vendor directory which holds dependencies. There's a separate script for bumping those but that's rarely used. The CI is green so the PR is good. Thanks.

**Comment by laurafitzgerald** [comment]
@soltysh One final check failing on `do-not-merge/hold` labels.

**Comment by soltysh** [comment]
> @soltysh One final check failing on `do-not-merge/hold` labels.

Yes, that's intentional since we're closing 4.1 and don't want any change to our temporarily. Only critical bugs are being let in.

**Comment by laurafitzgerald** [comment]
> > @soltysh One final check failing on `do-not-merge/hold` labels.
> 
> Yes, that's intentional since we're closing 4.1 and don't want any change to our temporarily. Only critical bugs are being let in.

Thanks for the info @soltysh Should I do anything else here then?

**Comment by soltysh** [comment]
@laurafitzgerald nope, just ping me in a week or so in case this doesn't move, I might forget about it

**Pull request merged**